rmw_opensplice_cpp/src/types.hpp

```diff
@@ -199,14 +200,17 @@ class OpenSplicePublisherListener : public DDS::PublisherListener
   std::atomic<size_t> current_count_;
 };
 
-struct OpenSpliceStaticPublisherInfo
+struct OpenSpliceStaticPublisherInfo : opensplice_static_event_info
```

Comment:
Let's be consistent and do `public opensplice_static_event_info` (even though it doesn't make a difference since `OpenSpliceStaticPublisherInfo` is a `struct`.

Reply:
Oh, never mind, doing `public opensplice_static_event_info` would make it inconsistent...